app/models/spree/app_configuration_decorator.rb

```diff
@@ -1,7 +1,7 @@
-Spree::AppConfiguration.class_eval do
-  # Sisow Preferences
-  preference :sisow_merchant_id, :string, :default => '2537407799'
-  preference :sisow_merchant_key, :string, :default => '0f9b49d384b4836c543f76d23a923e2cd2cfaec6'
-  preference :sisow_test_mode, :boolean, :default => true
-  preference :sisow_debug_mode, :boolean, :default => false
+Spree::AppConfiguration.class_eval do
+  # Sisow Preferences
+  preference :sisow_merchant_id, :string, :default => '2537407799'
+  preference :sisow_merchant_key, :string, :default => '0f9b49d384b4836c543f76d23a923e2cd2cfaec6'
```

Comment:
Line is too long. [97/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.